mibidata/tiff.py

```diff
@@ -212,7 +212,7 @@ def write(filename, image, sed=None, optical=None, ranges=None,
                 'channel.target': image.targets[i],
             })
             page_name = (285, 's', 0, '{} ({})'.format(
-                image.targets[i], image.masses[i]))
```

Comment:
What encode will do here with γ, β, μ, π? will it replace it with \\u03b2 \\u03b3 \\u03b4 \\u03c0? IUs taht what we want?

Reply:
What it does is that it converts it to bytes string, and when writing to mibitiff, it gets converted back to utf-8. 

If you enter these directly as strings, tifffile tries to convert them to ascii, and things catch on fire. 

I'm not 100% sure why it is trying to convert strings to ascii but does not do the same with byte strings, but it works somehow.

I will ask someone else from the instrument control team to look at this as well.

Reply:
@murataksoy thanks for explanation I think it makes sense.  

@rileysparsons I hold on with my approval waiting for your comments. You can approve yourself if you think it is fine.